docs/resources/azure_redis_cache.md

```diff
@@ -0,0 +1,119 @@
+  its('sku.family')                       { should cmp 'C' }
+  its('location')                         { should cmp 'southcentralus' }
+end
+```
+```ruby
+describe azure_redis_cache(resource_group: 'MyResourceGroup', name: 'inspec-compliance-redis-cache') do
+  it  { should exist }
+end
+```
+## Parameters
+
+| Name                            | Description                                                                      |
+|---------------------------------|----------------------------------------------------------------------------------|
+| resource_group                  | Azure resource group that the targeted resource resides in. `MyResourceGroup`    |
+| name                            | Name of the Azure Redis Cache to test.                                           |
+
+The parameter set should be provided for a valid query:
+- `resource_group` and `name`
+
+## Properties
+
+| Property                      | Description                                                      |
+|-------------------------------|------------------------------------------------------------------|
+| id                            | Resource ID.                                                     |
+| name                          | Redis Cache Name.                                                |
+| location                      | Redis Cache Location.                                            |
+| type                          | Resource type.                                                   |
+| tags                          | Resource tags.                                                   |
+| properties.sku.name           | The type of Redis cache to deploy. Valid values: (Basic, Standard, Premium).|
+| properties.sku.family         | The SKU family to use. Valid values: (C, P). (C = Basic/Standard, P = Premium).|
+| properties.sku.capacity       | The size of the Redis cache to deploy. Valid values: for C (Basic/Standard) family (0, 1, 2, 3, 4, 5, 6), for P (Premium) family (1, 2, 3, 4).|
+| properties.provisioningState  | Redis instance provisioning status.                               |
+| properties.redisVersion       | Redis version.                                                    |
+| properties.enableNonSslPort   | Specifies whether the non-ssl Redis server port (6379) is enabled.|
+
+For properties applicable to all resources, such as `type`, `name`, `id`, `properties`, refer to [`azure_generic_resource`](azure_generic_resource.md#properties).
+
+Also, refer to [Azure documentation](https://docs.microsoft.com/en-us/rest/api/redis/redis/get) for other properties available.
+Any attribute in the response may be accessed with the key names separated by dots (`.`).
+
+## Examples
+
+### Test <>
+```ruby
+describe azure_redis_cache(resource_group: 'MyResourceGroup', name: 'inspec-compliance-redis-cache') do
+  its('properties.provisioningState') { should eq 'Succeeded' }
+end
+```
```

Comment:
Is the value should 'standard' is correct or it needs to be 'Standard_1vCores'?

Reply:
Standard is the right one as per doc